base_geoengine/fields.py

```diff
@@ -170,6 +184,30 @@ class GeoLine(GeoField):
     type = 'geo_line'
     geo_type = 'LINESTRING'
 
+    @classmethod
+    def from_points(cls, cr, point1, point2):
+        """
+        Converts given points in parameter to a line.
+        :param cr: DB cursor
+        :param point1: Point (BaseGeometry)
+        :param point2: Point (BaseGeometry)
+        :return: LINESTRING HEX
+        """
+        sql = """
+        SELECT
+            ST_MakeLine(
+                ST_GeomFromText(%(wkt1)s, 4326),
```

Comment:
Srid was hardcoded in ```GeoPoint.from_latlon``` method, so I thought there was a reason for that. Should I change the hardcoded ```srid``` in this method too?